minishift-ci-index.yaml

```diff
@@ -75,3 +75,8 @@
           ci_project: '{name}'
           ci_cmd: '/bin/bash centos_ci.sh'
           timeout: '20m'
+      - '{git_repo}':
```

Comment:
Unrelated question, 'name: atomicapp' - is this what we use initially? Was there not talk about creating a new name/group for us?